ecs-cli/modules/utils/compose/convert_task_definition.go

```diff
@@ -530,3 +531,12 @@ func convertToULimits(cfgUlimits yaml.Ulimits) ([]*ecs.Ulimit, error) {
 
 	return ulimits, nil
 }
+
+// GoString returns deterministic string representation
+func SortedGoString(v interface{}) (string, error) {
```

Comment:
It's not clear to me where the sorting is happening.  Can you explain that a bit?